projects/plugins/backup/.gitattributes

```diff
@@ -17,7 +17,8 @@ composer.lock                               production-exclude
 gulpfile.babel.js                           production-exclude
 package.json                                production-exclude
 phpunit.xml.dist                            production-exclude
-readme.md                                   production-exclude
+README.md                                   production-exclude
```

Comment:
Seems odd to be excluding README.md from the mirror. We probably want it to show up at https://github.com/Automattic/jetpack-backup-plugin#readme.

Look at the unfriendly display at https://github.com/Automattic/jetpack-production#readme for a counterexample. We should fix (in a different PR, of course).

Reply:
Thanks Brad, changing L213 back seemed to have worked, not entirely sure what got mixed up there.

Regarding not showing `README.md` in the mirror repo, I was just following Jetack's lead there. If I understand correctly we'll need the `.txt` with the stable tag for releasing on WPorg, so it might be simpler to just maintain `readme.txt` in the mirror repo? @kraftbj any thoughts on this?

Reply:
Yeah, we need readme.txt for wporg. But GH displays readme.txt as plain text, not markdown.